src/GraphQL/Types/Collections/FieldFields.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace GraphQL.Types
+{
+    /// <summary>
+    /// A class that represents a set of fields for <see cref="IComplexGraphType"/> i.e <see cref="ComplexGraphType{TSourceType}"/>.
+    /// </summary>
+    public class FieldFields : IEnumerable<FieldType>
```

Comment:
```suggestion
    public class FieldList : IEnumerable<FieldType>
```
I think `FieldFields` is confusing.

Reply:
Changed to `TypeFields`.